Automattic/wp-calypso
# GS on Personal A/B: Neutralise style variation wording in theme details

We're changing the style variation wording to always say 'Custom styles' rather than 'Premium styles' to remove the association with the Premium plan. This will allow us to experiment with paywalling style variations behind different plans.



Related to https://github.com/Automattic/dotcom-forge/issues/2857

## Proposed Changes

Generalise some global styles wording on the theme details page

## Testing Instructions



1. Use calypso live link
2. Go through the /start process until you reach the design picker grid
3. Look at a theme with > 2 style variations
  * It should have a heading that says 'Custom Styles'
  * The text should say 'Unlock custom styles and tons of other features with the Premium plan, or try them out now for free.'
4. Look at a Premium theme with multiple style variations, it should just have a single heading that says 'Styles'
4. Look at a theme with two style variations
  * It should behave as above except it says 'Custom Style'
5. Go to /themes, repeat the steps above.
 
Before | After
-------|------
![image](https://github.com/Automattic/wp-calypso/assets/93301/6982f8e6-7ae2-44a4-80f8-6462d9b80698) | ![image](https://github.com/Automattic/wp-calypso/assets/93301/bcc77770-8446-4df5-a12c-75012799e6f1)


## Pre-merge Checklist
